mock-sequencer/wss.ts

```diff
@@ -0,0 +1,170 @@
+import { LOCAL_WS_PORT, REMOTE_WS_URL } from './consts'
+import WebSocket, { Server as WebSocketServer } from 'ws'
+
+export class MockSequencer {
+    constructor() {
+        this.wss = new WebSocketServer({ port: LOCAL_WS_PORT });
+        this.wss.on('connection', (ws) => {
+            this.clients.add(ws)
+            this.connectRemoteWs()
+            console.log('Client connected', this.clients.size)
+            ws.on('close', () => {
+                this.clients.delete(ws)
+            })
+        })
+
+    }
+
+    public getCurrentCount() {
+        return this.blockNumber
+    }
+
+    public setSkipNext() {
+        console.log('setting next block to be skipped', this.getCurrentCount() + 1)
+        this.skipNext = this.getCurrentCount() + 1
+    }
+
+    public setSendInRandom() {
+        this.sendInRandom = true
+    }
+
+    public setSendOversized() {
```

Comment:
what is the use case of this?

Reply:
This is used for the upcoming test where malicious sequencer send an oversized message to sequencer feed